package.json

```diff
@@ -1,77 +1,119 @@
-      "jmespath": "0.15.0",
-      "sax": "1.2.1",
-      "xml2js": "0.4.17",
-      "xmlbuilder": "4.2.1"
-    },
-    "main": "lib/aws.js",
-    "browser": {
-      "lib/aws.js": "./lib/browser.js",
-      "fs": false,
-      "./global.js": "./browser.js",
-      "./lib/node_loader.js": "./lib/browser_loader.js"
-    },
-    "browserify": { "transform": "./dist-tools/transform.js" },
-    "directories":  {
-      "lib": "lib"
-    },
-    "engines": {
-      "node": ">= 0.8.0"
-    },
-    "repository": {
-      "type" : "git",
-      "url": "git://github.com/aws/aws-sdk-js"
-    },
-    "bugs" : {
-      "url" : "http://github.com/aws/aws-sdk-js/issues",
-      "mail" : ""
-    },
-    "license": "Apache-2.0",
-    "keywords": [
-      "api", "amazon","aws","ec2","simpledb","s3","sqs","ses","sns","route53","rds","elasticache","cloudfront","fps",
-      "cloudformation","cloudwatch","dynamodb","iam","swf","autoscaling","cloudsearch","elb",
-      "loadbalancing","emr","mapreduce","importexport","storagegateway","workflow","ebs","vpc","beanstalk",
-      "glacier","kinesis","cloudtrail", "waf"
-    ],
-    "scripts" : {
-      "test" : "npm -s run-script lint && npm -s run-script unit && npm -s run-script buildertest && npm -s run-script browsertest && ([ -f configuration ] && npm -s run-script integration || true)",
-      "unit" : "istanbul `[ $COVERAGE ] && echo 'cover _mocha' || echo 'test mocha'` -- test test/json test/model test/protocol test/query test/services test/signers test/xml test/s3 test/cloudfront",
-      "coverage": "istanbul cover ./node_modules/mocha/bin/_mocha -- test test/json test/model test/protocol test/query test/services test/signers test/xml",
-      "browsertest": "rake browser:test",
-      "buildertest": "mocha --compilers coffee:coffee-script -s 1000 -t 10000 dist-tools/test",
-      "integration": "cucumber.js",
-      "lint" : "eslint lib dist-tools/*.js",
-      "console": "./scripts/console"
-    }
+  "name": "aws-sdk",
+  "description": "AWS SDK for JavaScript",
+  "version": "2.7.8",
+  "author": {
+    "name": "Amazon Web Services",
+    "email": "",
+    "url": "http://aws.amazon.com/"
+  },
+  "homepage": "https://github.com/aws/aws-sdk-js",
+  "contributors": [
+    "Loren Segal <lsegal@amazon.com>",
+    "Trevor Rowe <trevrowe@amazon.com>"
+  ],
+  "devDependencies": {
+    "@types/node": "^6.0.46",
+    "browserify": "13.1.0",
+    "chai": "*",
+    "coffee-script": "1.6.3",
+    "coffeeify": "*",
+    "coveralls": "2.x",
+    "cucumber": "0.5.x",
+    "eslint": "1.x",
+    "insert-module-globals": "^7.0.0",
+    "istanbul": "*",
+    "mocha": "*",
+    "repl.history": "*",
+    "semver": "*",
+    "typescript": "2.0.8",
+    "uglify-js": "2.x"
+  },
+  "dependencies": {
+    "buffer": "4.9.1",
+    "crypto-browserify": "1.0.9",
+    "jmespath": "0.15.0",
+    "querystring": "0.2.0",
+    "sax": "1.1.5",
+    "url": "0.10.3",
+    "uuid": "3.0.0",
+    "xml2js": "0.4.15",
+    "xmlbuilder": "2.6.2"
```

Comment:
whoops, my bad

When I merged in the changes from master, the entire `package.json` file got replaced and I neglected to update the versions